From badfe64569f6aaa2d28b6cc1f9e9f5fdc43e5347 Mon Sep 17 00:00:00 2001
From: tuxuser <462620+tuxuser@users.noreply.github.com>
Date: Sun, 24 Jul 2022 21:54:58 +0200
Subject: [PATCH 1/2] dependencies: Pin mbedtls to tagged version v3.1.0
 (#1494)

* dependencies: Pin mbedtls to tagged version v3.1.0

* Update CMakeLists.txt

Remove extra space

Co-authored-by: LordGrey <48840279+Lord-Grey@users.noreply.github.com>
---
 dependencies/CMakeLists-mbedtls.txt.in | 2 +-
 dependencies/CMakeLists.txt            | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/dependencies/CMakeLists-mbedtls.txt.in b/dependencies/CMakeLists-mbedtls.txt.in
index 12888d5b..76e177c7 100644
--- a/dependencies/CMakeLists-mbedtls.txt.in
+++ b/dependencies/CMakeLists-mbedtls.txt.in
@@ -14,7 +14,7 @@ include(ExternalProject)
 ExternalProject_Add(
         mbedtls
         GIT_REPOSITORY        "https://github.com/ARMmbed/mbedtls.git"
-        GIT_TAG               origin/master
+        GIT_TAG               "v3.1.0" # Bump versions manually if necessary, do not rely on origin/master to be stable
         BUILD_ALWAYS          OFF
         DOWNLOAD_DIR          "${DOWNLOAD_DIR}"
         SOURCE_DIR            "${SOURCE_DIR}"
diff --git a/dependencies/CMakeLists.txt b/dependencies/CMakeLists.txt
index 482b5e5a..f64b0d38 100644
--- a/dependencies/CMakeLists.txt
+++ b/dependencies/CMakeLists.txt
@@ -308,7 +308,7 @@ if(ENABLE_DEV_NETWORK)
 			FetchContent_Declare(
 				mbedtls
 				GIT_REPOSITORY        https://github.com/ARMmbed/mbedtls.git
-				GIT_TAG               origin/master
+				GIT_TAG               "v3.1.0" # Bump versions manually if necessary, do not rely on origin/master to be stable
 				BUILD_ALWAYS          OFF
 				GIT_PROGRESS          1
 				DOWNLOAD_DIR          "${MBEDTLS_DOWNLOAD_DIR}"
-- 
2.38.0

